apps/opik-backend/config.yml

```diff
@@ -65,3 +65,9 @@ server:
   enableVirtualThreads: ${ENABLE_VIRTUAL_THREADS:-false}
   gzip:
     enabled: true
+
+rateLimit:
+    enabled: ${RATE_LIMIT_ENABLED:-false}
+    generalEvents:
+      limit: ${RATE_LIMIT_GENERAL_EVENTS_LIMIT:-5000}
+      durationInSeconds: ${RATE_LIMIT_GENERAL_EVENTS_DURATION_IN_SEC:-1}
```

Comment:
Let's double check the values. Per the requirements ticket, I believe it should be 5K-10K per minute.

Reply:
You are right, sorry mixed it

Reply:
@thiagohora add a high default since its used for open source. we will set the limits in our comet.com installation.